### ISSUE TITLE
Provide moderation commands (ban, unban, kick) from Slack side

### ISSUE THREAD

**Issue by MurzNN** [matrix-org/matrix-appservice-slack#116]
Many Slack communities (eg Drupal https://www.drupal.org/project/infrastructure/issues/2885060) afraid to configure Matrix bridge for those channels, because they will lost moderation feature for room members on Matrix side: Slack channel admins will can't ban / unban / kick Matrix users from Slack interface.
For solve this issue will be good to provide some admin commands from Slack side, something like this:
```
!matrixbridge ban @spammer:matrix.org
```
So bridge bot will can check Slack permissions of this message author in current Slack channel, and if banning is allowed from him - do the ban action on Matrix side.

**Comment by MurzNN** [comment]
Or, if Riot Bridge Slack app already emulate Matrix users as bot-based virtual users on Slack side, we can hook ban/kick events on Slack side and convert them to Matrix events?

**Comment by Half-Shot** [comment]
We could do slash commands for these:

![image](https://user-images.githubusercontent.com/2072976/66405202-ad8b1f00-e9e1-11e9-9c24-84dfa2247b48.png)

**Comment by MurzNN** [comment]
Yes, good solution!

**Comment by MurzNN** [comment]
Other solution, provided by @tulir is reply to message with `/ban` text (for Telegram https://github.com/tulir/mautrix-telegram/issues/357). But replies on Slack are converted to threads, so maybe it will not so good for Slack.

**Comment by Half-Shot** [comment]
Yes, it's difficult to convey how that works to a user whereas slack users are used to slash commands.

**Comment by expipiplus1** [comment]
This was a concern raised in the AM32 Slack when chatting about bridging to Matrix.
